target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -525,6 +526,8 @@ ramips_setup_macs()
 	arcwireless,freestation5|\
 	lenovo,newifi-y1|\
 	lenovo,newifi-y1s|\
+	netgear,ex3700|\
+	netgear,ex6130|\
```

Comment:
Hmm, why is this added here? From my quick Google image search, both devices only have a single Ethernet port, which OpenWrt maps as LAN. So setting a WAN MAC address should not be necessary.

Reply:
It is even explicitly set up as `ucidef_set_interface_lan "eth0"` above.

Reply:
@blocktrron 
https://github.com/openwrt/openwrt/pull/2286#issuecomment-517123187

Have a look at this evaluation.